libcodechecker/log/log_parser.py

```diff
@@ -435,21 +436,39 @@ def get_compiler_standard(compiler, language):
         return standard
 
     @staticmethod
-    def set(details):
+    def load_compiler_info(filename, compiler):
+        contents = load_json_or_empty(filename, {})
+        compiler_info = contents.get(compiler)
+        if compiler_info is None:
+            LOG.error("Could not find compiler %s in file %s" %
```

Comment:
pylint did not warn about this log format? Did you rebase to the latest master?
python can do some logging optimization if the right format is used like this:
`LOG.error("Could not find compiler %s in file %s", compiler, filename)`

Reply:
Fixed, thanks.